components/BottomNavigation/examples/BottomNavigationTypicalUseExample.swift

```diff
@@ -0,0 +1,96 @@
+/*
+ Copyright 2017-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+import Foundation
+import MaterialComponents
+
+class BottomNavigationTypicalUseSwiftExample: UIViewController {
+
+  let appBar = MDCAppBar()
+
+  // Create a bottom navigation bar to add to a view.
+  let bottomNavBar = MDCBottomNavigationBar()
+
+  init() {
+    super.init(nibName: nil, bundle: nil)
+    self.title = "Bottom Navigation (Swift)"
+
+    self.addChildViewController(appBar.headerViewController)
+    let color = UIColor(white: 0.2, alpha:1)
+    appBar.headerViewController.headerView.backgroundColor = color
+    appBar.navigationBar.tintColor = UIColor.white
+    appBar.navigationBar.titleTextAttributes = [NSForegroundColorAttributeName : UIColor.white]
+    
+    commonBottomNavigationTypicalUseSwiftExampleInit()
+  }
+
+  required init?(coder aDecoder: NSCoder) {
+    super.init(coder: aDecoder)
+  }
+
+  func commonBottomNavigationTypicalUseSwiftExampleInit() {
+    view.backgroundColor = .lightGray
+    view.addSubview(bottomNavBar)
+
+    // Always show bottom navigation bar item titles.
+    bottomNavBar.titleVisibility = .always
+
+    // Cluster and center the bottom navigation bar items.
+    bottomNavBar.alignment = .centered
+
+    // Add items to the bottom navigation bar.
+    let tabBarItem1 = UITabBarItem(title: "Home", image: UIImage.init(named: "Home"), tag: 0)
```

Comment:
You can remove the `init` and just have `UIImage(name: "Home")`.

Try doing that for all `init`s

Reply:
Done.